spikingjelly/activation_based/ann2snn/recipes/sta_transformer.py

```
@@ -0,0 +1,1010 @@
+        "attention_mask",
+        "attn_mask",
+        "causal_mask",
+        "decoder_attention_mask",
+        "encoder_attention_mask",
+        "input_ids",
+        "key_padding_mask",
+        "labels",
+        "mask",
+        "position_ids",
+        "token_type_ids",
+    }
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)